### ISSUE TITLE
Display stopped working after latest kernel update

### ISSUE THREAD

**Issue by dbcodemania** [waveshare/7inch-DSI-LCD-C#6]
Display stopped working after kernel update to version 5.10.103. 
Can you please share the updated drivers or source code if possible?

**Comment by dbcodemania** [comment]
Display again stopped working after kernel update to version Linux rpdb01 5.15.30-v8+ #1536 SMP PREEMPT Mon Mar 28 13:53:14 BST 2022 aarch64 GNU/Linux.
@waveshare Can you please help with something so that we can avoid posting it again and again whenever there would be any kernel update?
May be you can share the source code so that we can compile it by ourselves.

**Comment by craCH** [comment]
Same problem here. Can you please share the source?

**Comment by morphias2004** [comment]
Yep.   Same issue - everything working found and now have to do a full reinstall.

This is beyond a joke!

The DFRobot DSI screen's don't require any specific drivers - they just work.

**Comment by morphias2004** [comment]
Run this and reboot:
`
sudo rpi-update b4145cfaa838049fcc1174d1311a98a854703c29`

It will roll back to Kernel 5.10.92.

**Comment by dbcodemania** [comment]
I tried to downgrade kernel version but original camera module stopped working due to firmware mismatch.
`sudo rpi-update 7a87d2221cbcd02b5948a5448717aaebab4a1aa3 #to downgrade to 5.10.103`
If we want to use this Waveshare screen  then we cannot execute the update commands `apt update` and `apt upgrade` which means we can use only the first release of Raspberry Pi OS.

**Comment by shubhamkastwar1** [comment]
:D I bought it quite recently and I thought my Pi went bed so i tried all my Pi's (2GB 4GB and 8GB) and eventually it was the script that was broken. So i will be stick with the old version.

@waveshare I hope you can update the code according to the latest update of raspberry pi OS
